server/database.js

```diff
@@ -0,0 +1,39 @@
+function dataEndpoint(pool, request, response) {
+  // const test_location = request.headers['x-teachermoments-location'];
+  // console.log(test_location)
+
+  const location = 'http://localhost:3000/teachermoments/turner?KevinTesting20180319';
+
+  const values = [location];
```

Comment:
This makes a lot of sense! I'll get started on working on things locally. Looking ahead, how exactly do I work with the production database? I don't think I've ever done that part before.

Reply:
Sounds good!  Here's the Heroku Postgres docs: https://devcenter.heroku.com/articles/heroku-postgresql.  Keep in mind that operations there are running on the production database, so if you run INSERT UPDATE or DELETE operations you will muck with real production data.  Running `SELECT` queries will just read and are generally much safer :)

Reply:
Alright, so I just added all of the extra tables I need to the production database. I've tested that my code does run as expected in a local development database. I also have a script to transcribe all of the emails in `latest-consent.json` into a psql database which I have tested locally.

Is there any particular reason you said to manually insert the emails? Should I not use the script?
>F. Take that set of emails, and manually run SQL comments to insert them into the production database.

Reply:
>G. Test and verify that works as expected.

Also, is there a way to test this works in production outside of merging the code? Do I just get rid of the NODE_ENV=development tag? That seems dangerous

Reply:
Awesome!  Your script is even better.

It changes the order of the steps here a bit, which is fine.  You'll have to deploy before you run the script.  So then when this is all ready, you can replace steps F and G with: a) deploy, b) verify that you can sign in as a researcher, but can't see any data because none of it is consented and then c) run your script and d) verify you can now see only consented data.

Reply:
hmmmmm can I deploy without merging this work into master?

Reply:
You can but if you want to deploy you should merge instead.  Here I mean "merge and deploy" :)